src/Command/StackCreateCommand.php

```diff
@@ -63,9 +64,11 @@ protected function askForOperation($operations, $input, $output) {
                     continue;
                 }
                 $options[$optionName] = $this->askForOption($optionName, $property['type'], $input, $output);
+            } else {
+                $options[$optionName] = $this->askForOption($optionName, $property['type'], $input, $output);
```

Comment:
No need to duplicate this assignent line. Just move it outside of the second `if`....  My bad